### PR TITLE
fix left_join and add test

### PR DESCRIPTION
**Pull request by hugolarzabal**

### CODE REVIEW

**Review by ggsdc — APPROVED** [review]

**Comment by ggsdc** [comment]
@gvalleg @AntonioGonzalezSuarez @dlopbar @l-prr could any of you review this PR as well?

File: mango/table/pytups_table.py [hunk 1]
```diff
@@ -235,31 +244,114 @@ def join(self, table2, by=None, suffix=None, jtype="full", empty=None) -> "Table
         :param empty: values to give to empty cells created by the join.
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_table.py:244]: Outdated docstring

File: mango/table/pytups_tools.py [hunk 2]
```diff
@@ -255,10 +263,27 @@ def left_join(table1, table2, by=None, suffix=None, empty=None):
     :param empty: values to give to empty cells created by the join.
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_tools.py:263]: Outdated docstring

File: mango/table/pytups_tools.py [hunk 3]
```diff
@@ -276,10 +301,27 @@ def right_join(table1, table2, by=None, suffix=None, empty=None):
...
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_tools.py:301]: Outdated doctring

File: mango/table/pytups_tools.py [hunk 4]
```diff
@@ -297,10 +339,27 @@ def full_join(table1, table2, by=None, suffix=None, empty=None):
...
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_tools.py:339]: Outdated docstring

File: mango/table/pytups_tools.py [hunk 5]
```diff
@@ -318,7 +377,16 @@ def inner_join(table1, table2, by=None, suffix=None, empty=None):
...
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_tools.py:377]: Outdated docstring

File: mango/table/pytups_table.py [hunk 6]
```diff
@@ -235,31 +244,114 @@ def join(self, table2, by=None, suffix=None, jtype="full", empty=None) -> "Table
...
+        table2,
+        by=None,
+        suffix=None,
+        empty=None,
+        if_empty_table_1=None,
+        if_empty_table_2=None,
+    ) -> "Table":
         """
         Shortcut to join(type="left")
         """
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_table.py:271]: Missing docstring

File: mango/table/pytups_table.py [hunk 7]
```diff
@@ -235,31 +244,114 @@ def join(self, table2, by=None, suffix=None, jtype="full", empty=None) -> "Table
...
+        table2,
+        by=None,
+        suffix=None,
+        empty=None,
+        if_empty_table_1=None,
+        if_empty_table_2=None,
+    ) -> "Table":
         """
         Shortcut to join(type="right")
         """
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_table.py:295]: Missing docstring

File: mango/table/pytups_table.py [hunk 8]
```diff
@@ -235,31 +244,114 @@ def join(self, table2, by=None, suffix=None, jtype="full", empty=None) -> "Table
...
+        table2,
+        by=None,
+        suffix=None,
+        empty=None,
+        if_empty_table_1=None,
+        if_empty_table_2=None,
+    ) -> "Table":
         """
         Shortcut to join(type="full")
         """
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_table.py:319]: Missing docstring

File: mango/table/pytups_table.py [hunk 9]
```diff
@@ -235,31 +244,114 @@ def join(self, table2, by=None, suffix=None, jtype="full", empty=None) -> "Table
...
+        table2,
+        by=None,
+        suffix=None,
+        empty=None,
+        if_empty_table_1=None,
+        if_empty_table_2=None,
+    ) -> "Table":
         """
         Shortcut to join(type="inner")
         """
```
Comment (AntonioGonzalezSuarez) [inline mango/table/pytups_table.py:343]: Missing docstring

**Review by AntonioGonzalezSuarez — CHANGES_REQUESTED** [review]

**Review by AntonioGonzalezSuarez — APPROVED** [review]

**Review by AntonioGonzalezSuarez — APPROVED** [review]
Approved

**Review by ggsdc — APPROVED** [review]

**Review by AntonioGonzalezSuarez — APPROVED** [review]

**Pull request merged**